src/main/java/jenkins/plugins/office365connector/WebhookJobPropertyDescriptor.java

```diff
@@ -27,12 +27,14 @@
 import net.sf.json.JSONObject;
 import org.apache.commons.lang.StringUtils;
 import org.kohsuke.stapler.QueryParameter;
+import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.StaplerRequest;
 
 /**
  * Job Property Descriptor.
  */
 @Extension
+@Symbol("office365Webhooks")
```

Comment:
This name is up for debate

Reply:
Can you change this to office365ConnectorWebhooks. This would be in sync with office365ConnectorSend

Reply:
No that's conflicting with the other syntax
This is used for the properties

Reply:
Oh I missed the connectorWebhooks 👍 I thought you said you wanted it to be named ConnectSend...